### PR TITLE
Implement Sign SC FE No Description

### PR DESCRIPTION
**Pull request by pivilartisant**
Screenshot 2023-08-24 at 11 15 41

### CODE REVIEW

File: wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx
[the remark this answers is not in the document]
Comment (pivilartisant) [reply wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx:1]: haha it was coming but i wasn't quick enought ^^

File: wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        </div>
+      </div>
+
+      <hr className="h-0.25 bg-neutral opacity-40 w-full" />
+
+      <div className="flex flex-col gap-2 w-full">
+        <div className="flex w-full items-center justify-between">
+          <p>Fees:</p>
+          {/* Fees are not implemented so I'm hardcoding it to design value */}
+          <p>1.000000000 nMAS</p>
```
[the remark this answers is not in the document]
Comment (pivilartisant) [reply wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx:56]: I think we are still determining how much fees are and how to caluculate them, this is standard in wallet, should I remove field altogether ?

File: wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+      <div className="flex flex-col gap-2 w-full">
+        <div className="flex w-full items-center justify-between">
+          <p>{Intl.t('password-prompt.sign.fees')}</p>
+          {/* Fees are not implemented so I'm hardcoding it to design value */}
+          <p>1.000000000 nMAS</p>
+        </div>
+        <div className="flex w-full items-center justify-between">
+          <p>{Intl.t('password-prompt.sign.coins')}</p>
+          <p>{Coins} MAS</p>
```
Comment (mazmassa) [inline wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx:60]:
```suggestion
          <p>{Coins} XMA</p>
```

Comment (Thykof) [reply wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx:60]: no

File: wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                    <div>Max Coins: {signData.MaxCoins}</div>
+                    <div>Max Gas: {signData.MaxGas}</div>
+                    <div>From: {maskAddress(signData.WalletAddress)}</div>
+                  </>
+                );
+              case 'Buy Roll':
+              case 'Sell Roll':
+                return (
+                  <>
+                    <div>Description: {signData.Description}</div>
```
Comment (Thykof) [inline wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx:102]: translation keys needed here

Comment (pivilartisant) [reply wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx:102]: This is going to be refactored into a component, so if you don't mind I would like to keep it like until the time is right ? :)

File: wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx (hunk 2 above)
Comment (pivilartisant) [reply wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx:60]: I think we decided that it was 'MAS' as token naming standard @longgamma-massa  can you confirm ?

**Comment by pivilartisant** [comment]
Screenshot 2023-08-24 at 10 52 11

**Review by Thykof — APPROVED** [review]

**Review by mazmassa — APPROVED** [review]

**Pull request merged**